dockstore-client/src/test/java/io/dockstore/client/cli/WesLauncherIT.java

```diff
@@ -57,7 +57,7 @@ public Workflow buildFakeWorkflow() {
         return workflow;
     }
 
-    public WorkflowClient mockWorkflowClient(String configPath) throws ApiException {
```

Comment:
Can't comment higher but should this class also be `@Category(WesTests.class)`? I forget if it runs by default in all jobs otherwise. 

Separately, system rule, mute for successful tests as well.

Reply:
This test class was improperly named, it isn't an integration test. Changed the file to `WesLauncherTest`.

Also muted on successful tests.

https://github.com/dockstore/dockstore-cli/pull/123/commits/e0013db5245e3273ba6af393660c5e27fb2b632f